packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -190,6 +193,67 @@ export const visualizeSharedString: VisualizeSharedObject = async (
 	};
 };
 
+/**
+ * Default {@link VisualizeSharedObject} for {@link TaskManager}.
+ */
+export const visualizeTaskManager: VisualizeSharedObject = async (
+	sharedObject: ISharedObject,
+	visualizeChildData: VisualizeChildData,
+): Promise<FluidObjectNode> => {
+	const taskManager = sharedObject as TaskManager;
+
+	const { id: fluidObjectId } = taskManager;
+
+	let tasks: string[] | undefined = taskManager.getTasks?.();
+
+	if (tasks === undefined) {
+		// Special backwards compat attempt to get the underlying task list out, for users missing the `getTasks` method.
+		// TODO (once `getTasks` is required): Remove this.
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+		const taskQueues = (taskManager as any).taskQueues as Map<string, unknown> | undefined;
+		if (taskQueues === undefined) {
+			// If we can't get to the underlying tasks, then we will simply note that we can't visualize this.
+			return visualizeUnknownSharedObject(taskManager, visualizeChildData);
+		}
+		tasks = [...taskQueues.keys()];
+	}
+
+	const children: Record<string, VisualChildNode> = {};
+	for (const taskId of tasks) {
+		const renderedChild: VisualTreeNode = {
+			children: {
+				assigned: {
```

Comment:
@jikim-msft I'm not sure if this is the visual approach we want or not. Here I broke the task list down into buckets, but we may instead just want a flat list with flags/icons indicating whether or not the task is queued / whether or not it is assigned to the "me" client"

Reply:
Scott will likely have thoughts on this too.